Client/src/Components/Graph/DatasetsList.tsx

```diff
@@ -0,0 +1,29 @@
+import { Box, Grid } from "@material-ui/core"
+
+import { DatasetRow } from "./DatasetRow"
+import { IDatasetModel } from "../../Models/Datasets/IDatasetModel"
+import React from 'react'
+
+interface IProps {
+  datasets: IDatasetModel[],
+  onRemoveDatasetClick: (datasetId: number) => void
```

Comment:
Can we move the interfaces elsewhere and import? Reduces duplicate code

Reply:
thing is this interface is internal to this component and is not meant to be reusable, we can have a clean convention that every component that has props, has an IProps interface inside of it at the top of the file. This interface is created specifically for the needs of this component. The interfaces that represent some data model or abstract idea have their own files and are exported as you can see in my IDatasetModel.ts file.